lib/rubocop/cop/rails/present.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cops checks for code that can be changed to `blank?`.
+      # Settings:
+      #   NotNilAndNotEmpty: Convert checks for not `nil` and `not empty?`
+      #                      to `present?`
+      #   NotBlank: Convert usages of not `blank?` to `present?`
+      #   UnlessBlank: Convert usages of `unless` `blank?` to `if` `present?`
+      #
+      # @example
+      #   # NotNilAndNotEmpty: true
+      #     # bad
+      #       !foo.nil? && !foo.empty?
```

Comment:
I'd remove the extra indentation under `bad`/`good`, as it's not something we typically do.